journal/modernized/32-epistles.adoc

```diff
@@ -929,7 +969,8 @@ such the Lord is honouring; everlasting praises to his name.
 
 My sincere love, in the unchangeable truth, is to you all, and my prayers to God for you.
 
-William Edmundson.
+[.signed-section-signature]
+William Edmundson
 
 === Concerning Men`'s and Women`'s Meetings.
```

Comment:
This should be a [.letter-heading] too... because it describes the contents of the following letter.